composer.json

```
@@ -125,7 +125,14 @@
         "seboettg/citeproc-php": "2.6.2"
     },
     "require-dev": {
-        "az-digital/az-quickstart-dev": "~1"
+        "az-digital/az-quickstart-dev": "~1",
+         "drush/drush": ">=12"
```

```suggestion
        "drush/drush": ">=12"
```
I think I'd prefer to keep the dev dependencies in the dev metapackage.  Maybe we should add the drush requirement there?

---

I might be misunderstanding the request here, but I do think that both of the Drush commands affected by this PR will be helpful for live sites and not just during development.